### ISSUE TITLE
binning="uniform" causes out of memory Out of memory in CreateRegressionBooster

### ISSUE THREAD

**Issue by ryanhausen** [interpretml/interpret#212]
If using `binning="uniform"`, `ebm.fit` raises a Memory error regardless of dataset size. I am using version 0.2.4.

Here is some code to reproduce the error:

``` python
import numpy as np
import pandas as pd
from interpret.glassbox import ExplainableBoostingRegressor

sample_data = pd.DataFrame(np.random.randint(0,100,size=(100, 4)), columns=list('ABCD'))

sample_x = sample_data.iloc[:, :-1]
sample_y = sample_data.iloc[:, -1]

ebm = ExplainableBoostingRegressor(
    binning="uniform",
    max_bins=256, 
    outer_bags=1,
    n_jobs=1
)
ebm.fit(x_train, y_train)
````

Am I calling the function incorrectly? 

Thanks for your help!

**Comment by interpret-ml** [comment]
Hi @ryanhausen, 

Thank you for reporting this and bringing it to our attention! Turns out we did have a bug here in a rare case where empty bins were generated for a feature. Uniform bins are more likely to fall into this condition, and we were able to use your code to reproduce and fix the problem. The fix is checked in to our develop branch, and should be resolved with the next release of interpret. 

We'll leave this issue open until the release goes out. Thanks again!

-InterpretML Team

**Comment by ryanhausen** [comment]
@interpret-ml thanks for the fix! I was wondering if you post your nightly build somewhere that I can pull and install? 

Thanks!

**Comment by interpret-ml** [comment]
Hi @ryanhausen,

We have wheels available from our build pipeline per commit, it's tricky to get to. It might be easier to build from source, either way. both options are provided below.

# Build and install from source

If you need the latest, building and installing from scratch would be:
`git clone https://github.com/interpretml/interpret.git && cd interpret/scripts && make install`

# Get the wheel per commit

To get wheels per commit, from our DevOps pipeline that should be accessible at:
https://dev.azure.com/ms/interpret/_build?definitionId=293

From there click on a commit, and you should see a similar page below. Click on the `[n] published` sub-header:
![image](https://user-images.githubusercontent.com/50031429/114250722-750c2e80-9953-11eb-9f73-47d64077a911.png)

After that, download the wheel artifact:
![image](https://user-images.githubusercontent.com/50031429/114250838-df24d380-9953-11eb-99a5-a3fb536ad5cd.png)

Let us know if you run into anything.

**Comment by interpret-ml** [comment]
Hi @ryanhausen, 

This should be fixed in the latest release of interpret (v0.2.5). You can use pip to upgrade now with `pip install -U interpret`. Thanks again for reporting this and helping us catch this bug!

-InterpretML Team

**Comment by ryanhausen** [comment]
Thanks so much!
